### PR TITLE
docs: clean up product roadmap

### PR DESCRIPTION
**Pull request by dave-at-koor**
Re-wrote the road map.

### CODE REVIEW

File: docs/development/product-roadmap.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+* Diagnostic scripts
+  * Gathers debugging information that is useful for troubleshooting issues in a Rook Ceph cluster
+* Extended Ceph Exporter
+  * Extends metrics exported from Promethius for improved utilization charts
+
+### Releases
+
+* [Koor Operator v0.2.0](https://github.com/koor-tech/koor-operator/releases/tag/koor-operator-0.2.0)
+* [Extended Ceph Exporter v1.2.8](https://github.com/koor-tech/extended-ceph-exporter/releases/tag/extended-ceph-exporter-1.2.8)
+* [Koor gather info scripts](https://github.com/koor-tech/koor-gather-info)
```
Comment (zalsader) [inline docs/development/product-roadmap.md:27]: We should probably start versioning this

File: docs/development/product-roadmap.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  * Rook Ceph cluster visualization
+  * See your data cluster configuration in a browser
+  * Click on each component to see configuration and operational characteristics
+* Storage System Builder
+  * Start from one of the basic templates that match common use cases
+  * Adjust configuration options through a GUI
+  * One-click cluster provisioning and adjustments
+  * Integrates with Koor Dashboard
+* Security Tools
+  * Set up SSO access to underlying Ceph cluster
```
Comment (zalsader) [inline docs/development/product-roadmap.md:52]: Isn't this ready?

**Review by zalsader — APPROVED** [review]
looks good

**Review by Javlopez — APPROVED** [review]

File: docs/development/product-roadmap.md (hunk 1 above)
Comment (dave-at-koor) [reply docs/development/product-roadmap.md:27]: agreed

File: docs/development/product-roadmap.md (hunk 2 above)
Comment (dave-at-koor) [reply docs/development/product-roadmap.md:52]: We have instructions in the Knowledge Center. For this item, I am imagining something more automated, where the IT person enters a few fields and clicks a button.

**Review by galexrt — APPROVED** [review]

**Pull request merged**